scrapy/http/request/json_request.py

```diff
@@ -13,7 +14,10 @@
 
 class JSONRequest(Request):
     def __init__(self, *args, **kwargs):
-        dumps_kwargs = kwargs.pop('dumps_kwargs', {})
+        dumps_kwargs = copy.deepcopy(kwargs.pop('dumps_kwargs', {}))
+        dumps_kwargs['sort_keys'] = True
```

Comment:
Hey, I think it is better to have .setdefault here, i.e. allow user to override it. I don't think it would harm, and there can be weird cases where disabled sort_keys is needed (e.g. user wants a specific field order, and relies on Python 3.7 dict key order to preserve it).

Reply:
Hey @kmike it is changed

Reply:
👍